### PR TITLE
Themable Webcomponent header

### PR DESCRIPTION
**Pull request by f-necas**
# Themable Webcomponent header

Main PR :
- https://github.com/georchestra/georchestra/pull/4120

This PR adds missing 'logo-url' attribute and new 'stylesheet' attribute

### CODE REVIEW

File: default.properties [hunk 1]
```diff
@@ -137,4 +147,4 @@ smtpPort=${SMTPPORT}
 # console will add all links to analytics and also execute XHR requests
 # header will diplay analytics app
 # default: true
-# analyticsEnabled=true
+analyticsEnabled=false
```
Comment (tkohr) [inline default.properties:150]: Has this been changed on purpose (as it is still `true` in the comment)?

Comment (f-necas) [reply default.properties:150]: Oups

File: default.properties [hunk 2]
```diff
@@ -37,6 +37,16 @@ useLegacyHeader=false
 # default /header/
 headerUrl=/header/
 
+# Stylesheet used to override default colors of header
+# More design can be set by overriding default classes & styles
+# Default empty string
+#georchestraStylesheet=http://localhost:3000/tmp.css
```
Comment (tkohr) [inline default.properties:43]: Not of great importance, but `style.css` would probably be more explicit naming.

File: default.properties [hunk 3]
```diff
@@ -37,6 +37,16 @@ useLegacyHeader=false
...
+
+# Logo URL
+# Used to set header's logo.
+logoUrl=https://www.georchestra.org/public/georchestra-logo.svg
```
Comment (tkohr) [inline default.properties:47]: As already mentioned, it would be nice to be able to point to a local resource here in the future.

**Review by tkohr — APPROVED** [review]
Thanks @f-necas !

**Pull request merged**